sklearn/metrics/tests/test_classification.py

```diff
@@ -1463,5 +1463,7 @@ def test_brier_score_loss():
     assert_raises(ValueError, brier_score_loss, y_true, y_pred + 1.)
     assert_raises(ValueError, brier_score_loss, y_true, y_pred - 1.)
     # calculate even if only single class in y_true (#6980)
-    assert_almost_equal(brier_score_loss([0], [0.5]), 0.25)
```

Comment:
@nelson-liu Considering the other tests that I had added, I felt these were redundant. I can add them back if you want.